pool/web/view/component/component_input.ml

```diff
@@ -749,7 +733,14 @@ let multi_select
   in
   div
     ~a:[ a_class (Elements.group_class classnames orientation) ]
-    ([ label [ txt (Elements.input_label language group_field None required) ]
+    ([ label
```

Comment:
Add contact hint: add hint: some experiments choose to send invitations in a different language, disregarding your contact language